tests/test_wave_01.py

```diff
@@ -1,10 +1,11 @@
+from logging import exception
```

Comment:
Looks like this module got imported - maybe by a VS Code unhelpful autocomplete? Feel free to delete them if not used.